mlir/lib/Dialect/LLVMIR/IR/LLVMDialect.cpp

```diff
@@ -62,14 +62,23 @@ static auto processFMFAttr(ArrayRef<NamedAttribute> attrs) {
   return filteredAttrs;
 }
 
+static auto processIntArithAttr(ArrayRef<NamedAttribute> attrs) {
+  SmallVector<NamedAttribute, 8> filteredAttrs(
+      llvm::make_filter_range(attrs, [&](NamedAttribute attr) {
+        return attr.getName() != "arithFlags";
```

Comment:
`getXYZAttrName` instead of hardcoding string.

Reply:
I agree this is much better, but it isn't actually a static method of the interface (although it is declared as such in tablegen). One needs a concrete operation type to get the attribute name. I will try casting the Operation argument to printLLVMOpAttrs to the interface to get a concrete instance.

Do you want me to also fix `processFMFAttr` while I am at it?

Reply:
Looking more into code, `StaticInterfaceMethod` seems to means it static per op type, not per interface, so this is just an unfortunate name choice, I guess. I see you've removed this function in latest commit so this is resolved.

Reply:
Ah sorry. I was misguided by "static" then...